docs/datamodel.rst

```diff
@@ -0,0 +1,615 @@
+                        + sagseventid : UUID\l
+                        + registreringfra : Datetime\l
+                        + registreringtil : Datetime\l
+                        + beskrivelse : String\l
+                }"
+            ]
+
+            Sagseventinfo_html [
+                fillcolor = lightpink
+                style = filled
+                label = "{Sagseventinfo_html|\l
+                        + objectid : Integer\l
+                        + html : String\l
+                        + sagseventinfoid : Integer\l
+                }"
+            ]
+
+            Sagseventinfo_materiale [
+                fillcolor = lightpink
+                style = filled
+                label = "{Sagseventinfo_materiale|\l
+                        + objectid : Integer\l
+                        + sti : String\l
+                        + md5sum : String\l
+                        + sagseventinfoid : Integer\l
+                }"
+            ]
+
+            Eventtype [
+                fillcolor = palegreen
+                style = filled
+                label = "{Eventtype|\l
+                        + objectid : Integer\l
+                        + beskrivelse : String\l
+                        + event : String\l
+                        + eventtypeid : Integer\l
+                }"
+            ]
+
+            Sagsevent -> Sag;
+            Sagsevent -> Fikspunktregisterobjekt;
+            Sagsinfo -> Sag;
+            Sagseventinfo -> Sagsevent;
+            Sagseventinfo_html -> Sagseventinfo;
+            Sagseventinfo_materiale -> Sagseventinfo;
+            Eventtype -> Sagsevent;
+        }
+
+Der findes en række sagsevents i FIRE. I tabellen herunder er de alle kort beskrevet.
+
+=========================  ============================================================================
+**Type**                   **Beskrivelse**
+-------------------------  ----------------------------------------------------------------------------
+``koordinat_beregnet``     Bruges når koordinater indsættes efter en beregning. Vil typisk resulterere
+                           i indsættelse af *n* koordinater og 1 beregning.
+``koordinat_nedlagt``      Bruges når en koordinat nedlægges.
+``observation_indsat``     Indsættelse af en eller flere observationer.
+``observation_nedlagt``    Bruges når en observation aflyses fordi den er fejlbehæftet
+``punktinfo_tilføjet``     Bruges når der tilføjes Punktinfo til et eller flere punkter
+``punktinfo_fjernet``      Bruges når Punktinfo fjernes fra et eller flere punkter
+``punkt_oprettet``         Bruges når et punkt og tilhørende geometri oprettes
+``punkt_nedlagt``          Bruges når et punkt og tilhørende geometri nedlægges. Bemærk at når et punkt
+                           nedlægges, afregistreres desuden alle tilknyttede koordinater, observationer
+                           og punktinformationer, da disse ikke længere har et tilhørsforhold.
```

Comment:
Er sådan en afregistrering noget, der er reversibelt? Umiddelbart læser jeg det ikke som sådan..

Reply:
Korrekt, det kan ikke laves om. Men skulle man gøre det ved en fejl kan man godt indlæse en ny koordinat. Så det er ikke sådan at nedlægger en koordinat at man udelukker muligheden for flere koordinater af den bestemte type på det specifikke punkt.